R/screenshot.R

```diff
@@ -39,6 +40,17 @@ chromote_session_screenshot <- function(
     expand <- rep(expand, 4)
   }
 
+  stopifnot(
+    "`options` must be a list" = rlang::is_list(options),
+    "`options` must be named" = rlang::is_named2(options())
+  )
```

Comment:
I think we'd need to add a `R >= 3.6` requirement?

Reply:
No, this is backwards compatible so older versions of R just get the old `stopifnot()` message

Reply:
Ah right, sorry I keep forgetting 👍